crates/noirc_frontend/src/lexer/lexer.rs

```diff
@@ -102,7 +102,8 @@ impl<'a> Lexer<'a> {
             Some('}') => self.single_char_token(Token::RightBrace),
             Some('[') => self.single_char_token(Token::LeftBracket),
             Some(']') => self.single_char_token(Token::RightBracket),
-            Some('"') => Ok(self.eat_string_literal()),
+            Some('"') => Ok(self.eat_string_literal(false)),
+            Some('f') => self.eat_format_string_literal(),
```

Comment:
Lets rename this so it doesn't look like we only check for fmt strings. Maybe `eat_format_string_or_alpha_numeric`.

Reply:
done